ffwd-client/src/main/java/com/spotify/ffwd/v1/Metric.java

```diff
@@ -80,7 +62,7 @@ public Metric(
     this.has = has;
```

Comment:
@lmuhlha @sjoeboo 
I don't see has() method in this class to set that field and I have a suspicion we are not using it. 
I for sure knew about "proc" field and it's gone. I think we could clean it.

Reply:
It is probably safe to keep it  for now.
 In the context of Metric class, it is used to determine which attribute is set.

Reply:
got it. I didn't see that. 👍